data/sheragi outfits.txt

```diff
@@ -0,0 +1,213 @@
+	category "Engines"
+	cost 80000000
+	thumbnail "outfit/fusiondrive"
+	"mass" 480
+	"outfit space" -480
+	"engine capacity" -230
+	"jump speed" 0.2
+	"hyperdrive" 1
+	"energy generation" 54
+	"heat generation" 156
+	"turn" 1920.0
+	"turning energy" 3.5
+	"turning heat" 9.5
+	"thrust" 190
+	"thrusting energy" 11
+	"thrusting heat" 33
+	"reverse thrust" 32
+	"reverse thrusting energy" 3.5
+	"reverse thrusting heat" 9.5
+	"flare sprite" "effect/fdrive"
+		"frame rate" 30
+		"rewind"
+	"flare sound" "atomic huge"
+	unplunderable 1
+	description `The Sheragi engines are much more than simple tubes spitting fire; they are a collection of interdependent devices, providing increased efficiency at the cost of modularity. This includes a hyperdrive, a huge fusion reactor, and the most powerful plasma engines you've ever seen. All simple technology, but enhanced to their extremes. The engines are fundamentally connected to the fusion reactor, increasing their thrust even further and thereby allowing the Emerald Sword to fly at impressive speeds and travel interstellar space with relative ease.`
+
+outfit "Sheragi Cooling Unit"
+	category "Systems"
+	cost 1000000
+	thumbnail "outfit/sheragicooling"
+	"mass" 40
+	"outfit space" -40
+	"active cooling" 142
+	"cooling energy" 16.2
+	description `This Sheragi cooling system is primitive at best, but it is active rather than passive, allowing it to dissipate much more heat than a simple radiator would. It drains a lot of power when subjected to the heat of battle, though.`
+
+outfit "Damaged Sheragi Systems"
+	category "Special"
+	cost 0
+	thumbnail "outfit/unknown"
+	"shields" -2000
+	"hull" -6000
+	"energy consumption" 3
+	"heat generation" 4
+	"jump drive" -1
+	unplunderable 1
+	description "There is a lot of broken Sheragi hardware in here. You need to fix that before you can take the ship back into Human space."
+
+
+# Weapons
+
+outfit "Dragonflame Cannon"
+	category "Secondary Weapons"
+	cost 150000000
+	thumbnail "outfit/dragonflame"
+	"mass" 156
+	"outfit space" -156
+	"weapon capacity" -156
+	"required crew" 10
+	"gun ports" -1
+	weapon
+		sound "dragonflame" 
+		icon "icon/dragonflame"
+		"fire effect" "dragonflame"
+		"hit effect" "bright hit"
+		"hit effect" "dragonflame plasma" 30
+		"hit effect" "dragonflame debris" 50
+		"velocity" 2000
+		"lifetime" 1
+		"reload" 180
+		cluster
+		"firing energy" 16000
+		"firing heat" 40000
+		"firing force" 1000
+		"shield damage" 17000
+		"firing fuel" 300
+		"hull damage" 9000
+		"piercing" 0.3
+		"heat damage" 40000
+		"hit force" 2000
+	description "This ancient Sheragi superweapon is simple in concept, but the cost involved in developing something like it is prohibitive, and it's hard to imagine what could motivate the creation of such a thing. It uses an obscene amount of energy to create an extremely powerful fusion reaction, and directs most of the energy of the explosion forwards in a devastatingly powerful beam of gamma radiation."
```

Comment:
description "The ancient Sheragi superweapon is simple in concept, but the cost involved in developing something like it is exorbitant. It's hard to envision what could have motivated the creation of such a thing. It uses an obscene amount of energy to create an extremely powerful fusion reaction, directing most of the energy of the resulting explosion forwards in a devastatingly powerful beam of gamma radiation."